cdap-data-fabric/src/test/java/co/cask/cdap/data2/dataset2/DatasetClassRewriterTest.java

```diff
@@ -436,6 +441,11 @@ public void enforce(EntityId entity, Principal principal, Set<Action> actions) t
     }
 
     @Override
+    public Set<? extends EntityId> isVisible(Set<? extends EntityId> entityIds, Principal principal) throws Exception {
+      throw new UnsupportedOperationException("This method needs to be implemented!");
```

Comment:
I think based on the createFilter() implementation we can just return entityIds here?